tests/unit/test_base.py

```diff
@@ -58,6 +58,13 @@ def test_to_excel():
     os.remove("testbase.xlsx")
 
 
+def test_to_parquet():
```

Comment:
Are you checking here for a 'file not found'? Please add a test to your `try except` block.